### DISCUSSION TITLE
Next update

### DISCUSSION THREAD

**Discussion by venkat2305** [deniscerri/ytdlnis#19] [Q&A] [answered]
What are your planned features for the next major update ?
Do you plan to create a telegram or discord community.

**Comment by deniscerri** [comment, marked as answer]
I think i need to rewrite a good chunk of this app. This was my first android project so i threw stuff together till they worked. 
I need to create a structure before thinking of big changes.
I'll try to find time since im still a student and working, so im doing this on my free time.

- Im thinking of converting the whole app to kotlin or jetpack compose
- download scheduling
- create a dialog on share menu
- maybe download multiple files at the same time
- reordering items in a download queue
- maybe make the app search on different sites and not just youtube, i think yt-dlp supports some sites or i can use other apis like i found with youtube

Idk maybe other stuff people could ask. I'm not a pro so im learning as i go. :)
I made a discord server btw:
https://discord.gg/WW3KYWxAPm
